hooks/pre/30-upgrade.rb

```diff
@@ -164,6 +171,7 @@ def fail_and_exit(message)
     upgrade_step :mark_qpid_cert_for_update
     upgrade_step :migrate_candlepin, :run_always => true
     upgrade_step :remove_gutterball
+    upgrade_step :delete_event_queue
     upgrade_step :start_tomcat, :run_always => true
```

Comment:
That doesn't answer why we're deleting the queue in a hook

Do you need to delete the queue for it to get setup correctly again?

If that's the case, it should be the job of Puppet to detect that things are misconfigured and recreate the queue with the right settings.

That could mean exec over here https://github.com/Katello/puppet-katello/blob/master/manifests/qpid.pp#L14